web/themes/custom/bglobal/source/_patterns/01-molecules/hero/_hero.scss

```diff
@@ -0,0 +1,74 @@
+.hero__title.h2 {
+  font-size: 3rem;
+  line-height: 5rem;
+}
+
+.hero__subtitle {
+  line-height: 1.5;
+}
+
+.hero__container {
+  background-color: #EEEEEE;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 450px;
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  align-content: center;
+  justify-content: center;
+}
+
+.hero__image__container {
+  grid-area: 1 / 1 / 1 / 4;
+}
+
+img.hero__image {
+  display: grid;
+  height: 100%;
+  width: 100%;
+  min-width: 350px; 
+  max-width: 100% !important;
```

Comment:
Since we don't have `img.hero__image` elsewhere, `!important` is probably not necessary here, as it shouldn't be overriding an existing style. With us creating classes specific to each component, it should help in avoiding the usage of `!important`.